android/android-email-sms.html

```diff
@@ -0,0 +1,320 @@
+<div class="snippet" id="firefox-android-sms-email-{{ snippet_id }}">
+  <style text="text/css">
+     div.switch-{{ snippet_id }}#brandLogo {
+         background-size: auto;
+     }
+
+     .switch-{{ snippet_id }}#brandLogo {
+         height: 192px;
+         width: 192px;
```

Comment:
Just a note here: Usually on logo replacement snippets we allow `brandLogo.height` and `brandLogo.width` to be variables, otherwise the replacement logo has to be `192x192`.